packages/overlay/src/overlay-stack.ts

```diff
@@ -22,19 +22,11 @@ function hasModifier(event: MouseEvent): boolean {
 }
 
 export class OverlayStack {
-    public overlays: ActiveOverlay[] = [];
```

Comment:
We can just use the DOM as our data structure. Follow the DRY principle and get rid of this.